### DISCUSSION TITLE
Using App Check with AngularFire

### DISCUSSION THREAD

**Discussion by gillarf** [angular/angularfire#3194] [Q&A]
Hi everyone

We'd really like to use Firebase App Check in our iOS and Android apps, built using AngularFire, but I can't find any documentation on it.

Does anyone have an examples of an AngularFire implementation?

Thanks very much for any help with this.

Kevin

**Comment by Jonathan-de-Kock** [comment]
I have also been looking for documentation regarding this!
It would be great to not have to use the normal firebase JS SDK to run appCheck alongside the AngularFire implementation!

Thanx for all the great work guys!

Regards
Jono

**Comment by andrelombaard** [comment]
I agree, also looking for this feature.

**Comment by driko-development** [comment]
It seems there is no folder for app check under compat. Does this mean it does not work yet in the latest version?
